### PR TITLE
Fibonacci  Series in java

### PR DESCRIPTION
**Pull request by ravivarmalearner**

### CODE REVIEW

File: Algorithms/Maths/Fibonacci series/Java/Fibonacci.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+class Fibonacci {
+    public static void main (String args[]) {  
+        int count, firstValue = 0, secondValue = 1, nextValue;
+        Scanner sc = new Scanner(System.in);
+        
+        // Input number of terms from user 
+        System.out.println("Enter the number of terms:\n");
+        count = sc.nextInt();
```
Comment (ravivarmalearner) [inline Algorithms/Maths/Fibonacci series/Java/Fibonacci.java]: // Storing user input

**Comment by ravivarmalearner** [comment]
I have added Fibonacci algorithm in java. please review it and let me know.

**Comment by jenistenxavier** [comment]
@ravivarmalearner : : It would be great if you can add the description along with the issue number.
Feel free to use our contribution guidelines (https://github.com/fnplus/interview-techdev-guide/blob/master/CONTRIBUTING.md) for reference

 Please resolve the conflicts and submit for review. Thanks.

**Comment by xlogix** [comment]
Thanks for contributing!

**Pull request merged**